server/channels/store/sqlstore/channel_store.go

```diff
@@ -1489,24 +1489,38 @@ func (s SqlChannelStore) GetDeletedByName(teamId string, name string) (*model.Ch
 	return &channel, nil
 }
 
-func (s SqlChannelStore) GetDeleted(teamId string, offset int, limit int, userId string) (model.ChannelList, error) {
+func (s SqlChannelStore) GetDeleted(teamId string, offset int, limit int, userId string, includePrivate bool) (model.ChannelList, error) {
```

Comment:
`includePrivate` is not quite the right term, is it? Even non-admins can see private channels if they are a member. Is there a better term? Or can we add a comment explaining what the parameter does?

Reply:
that could be ignoreTeamMembership or has[or can]ManageSystem but I'm not good at naming things ;)

Reply:
Ok I've settled on skipTeamMembershipCheck tell me if that seems a better term